app/Registration.php

```diff
@@ -69,6 +70,28 @@ public function centre()
         return $this->belongsTo('App\Centre');
     }
 
+    /**
+     * Get the first un-disbursed bundle on a Registration for any Centre.
+     * There should only be one... else make one if there are none.
+     *
+     * @return Model
+     */
+    public function currentBundle()
+    {
+        return $this->bundles()
+            ->where('disbursed_at', null)
+            ->orderBy('id', 'asc')
+            ->firstOrCreate(
+                [
+//                    "allocating_centre_id" =>  Auth::user()->centre,
+                    "registration_id" => $this->id,
+                ],
+                [
+                    "entitlement" => $this->family->entitlement,
```

Comment:
Do you need to catch if $this->family is null?

Reply:
entitlement should never be null - it's a calculated attribute that returns an int (i think)

Reply:
Just that you'd get a 500 if there is no family for bug reasons.